### PR TITLE
Rubocop rspec for testing

### PR DESCRIPTION
**Pull request by jeffcstock**
@thaisamarinho @mrreubenjones, you lovely people:

I've added rubocop rspec gem and then fixed all tests that were failing because of it. Basically rubocop rspec checks also the rspec files and we've added config files too. Now all tests are passing, but please have a look to double check.

### CODE REVIEW

File: config/initializers/app_keys.rb [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
+ENV['email_user_name'] = 'answerawesome'
+ENV['email_password']  = 'Sup3r$ecret$$'
 ENV['TWITTER_CONSUMER_KEY'] = 'ftCIdGCfv1w9cUeo1MNLYc2xd'
-ENV['TWITTER_CONSUMER_SECRET'] = 'oEO3cHGR9noj5j5sgNBNIW8YU1ufuiU5trfRt8HgGcJF0Y4y3P'
+ENV[
+  'TWITTER_CONSUMER_SECRET'
+  ] = 'oEO3cHGR9noj5j5sgNBNIW8YU1ufuiU5trfRt8HgGcJF0Y4y3P'
 
 ENV['FACEBOOK_APP_ID'] = '2244085749149962'
 ENV['FACEBOOK_APP_SECRET'] = '1ce1e1f0639eff17086811b44e2dbbc2'
```
Comment (thaisamarinho) [inline config/initializers/app_keys.rb]: Something is really wrong here bc this file should not be going up to github.
we need to make sure to put it on gitignore.

**Review by thaisamarinho — COMMENTED** [review]
Jeff, it looks wonderful to me, the only problem i see, and it's a big one, is the app_keys being pushed up to github. they should be in the gitignore file. I think they are there, but something is going wrong with it, could you please check that? 
Thank you! Great job here.

File: config/initializers/app_keys.rb (hunk 1 above)
Comment (jeffsmw) [inline config/initializers/app_keys.rb]: I'm not sure why that got included, I'll remove it right away

Comment (jeffcstock) [reply config/initializers/app_keys.rb]: I fixed this, so now i'll tag @whilestevego for a final check.

**Review by whilestevego — APPROVED** [review]
Looks good! Merge it once you solve the conflict. It's related to the `app_keys.rb` make sure that the file is deleted after the conflict is resolved.

**Pull request merged**